includes/wikia/traits/TitleTrait.php

```diff
@@ -152,4 +152,10 @@ public function getIndirectLinks( $level = 0 ) {
 
 		return $rows;
 	}
+
+	public function isTemplateDraft() {
+		global $wgEnableTemplateDraftExt;
+
+		return !empty( $wgEnableTemplateDraftExt ) && TemplateDraftHelper::isTitleDraft( $this );
```

Comment:
I wouldn't say that's the best approach as ``isTitleDraft`` uses messages from TemplateDraft extension and checking if global wgEnableTemplateDraftExt for every TemplateDraft internal usage doesn't make sens as you already are a part of extension.
I see it better as it is above.